book/02-git-basics/sections/recording-changes.asc

```diff
@@ -174,14 +174,14 @@ Changes not staged for commit:
 ----
 
 //////////////////////////
-The `CONTRIBUTING.md` file appears under a section named ``Changes not staged for commit'' – which means that a file that is tracked has been modified in the working directory but not yet staged.
+The `CONTRIBUTING.md` file appears under a section named "`Changes not staged for commit`" -- which means that a file that is tracked has been modified in the working directory but not yet staged.
 To stage it, you run the `git add` command.
 `git add` is a multipurpose command – you use it to begin tracking new files, to stage files, and to do other things like marking merge-conflicted files as resolved.
-It may be helpful to think of it more as ``add this content to the next commit'' rather than ``add this file to the project''.(((git commands, add)))
+It may be helpful to think of it more as "`add this content to the next commit`" rather than "`add this file to the project`".(((git commands, add)))
```

Comment:
```suggestion
It may be helpful to think of it more as ``add this content to the next commit'' rather than ``add this file to the project''.(((git commands, add)))
```